website/content/docs/agent/config/config-files.mdx

```diff
@@ -870,9 +870,9 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
       [`acl.tokens.initial_management`](#acl_tokens_initial_management).**
 
     - `default` ((#acl_tokens_default)) - When provided, the agent will
-      use this token when making requests to the Consul servers. Clients can override
-      this token on a per-request basis by providing the "?token" query parameter.
-      When not provided, the empty token, which maps to the 'anonymous' ACL token,
+      use this token when making requests to the Consul servers.
+      Clients can [override this token on a per-request basis](/api-docs#authentication).
+      When not overridden, the empty token, which maps to the 'anonymous' ACL token,
```

Comment:
```suggestion
      If the `default` token is not configured and no alternate token is passed in the authorization header, the empty token, which maps to the `anonymous` ACL token,
```
Correct? This is about the `default` parameter, so we should use that as part of the context.

Reply:
I reworded this whole section to hopefully make the interaction between the request-specific token, `default` token, and anonymous token clearer. Let me know what you think.